### ISSUE TITLE
Invalid Bundle

### ISSUE THREAD

**Issue by dima2394** [myTrackerSDK/mytracker-ios-spm#5]
Xcode 15.3
Invalid Bundle. The bundle xxxx.app/Frameworks/MyTrackerSDK.framework does not support the minimum OS Version specified in the Info.plist. (ID: 5d23175a-0308-4216-8489-237229c1ac18)
при попытки собрать архив - выдает такую ошибку. много попробовал всего для решения, но ничего не помогает. хэлп....(

**Comment by aserrr** [comment]
Добрый день,
Какая минимальная поддерживаемая версия iOS у приложения?
MyTracker SDK поддерживает 12.4+

**Comment by dima2394** [comment]
добрый день !
минимальная 17.0)

**Comment by aserrr** [comment]
Проверил на пустом приложении под iOS 17 с установкой зависимости через SPM
Archive работает корректно.
Пожалуйста, напишине в [поддержку](https://app.tracker.my.com/support/), возможно понадобится больше деталей чтобы разобраться в проблеме.

**Comment by crownvic** [comment]
Аналогичная проблема на Xcode 15.3 последнем, на Xcode 15.2 собирает нормально. Но это только при заливании в app store connect. Локально прекрасно всё собирается и работает на устройствах и симуляторах, но как только пытаешься релиз залить в стор, сразу ошибка вот эта при анализе.

**Comment by alysenko912** [comment]
Тоже столкнулся с данной проблемой. MinimumOSVersion приложения - 14.5, а у MyTrackerSDK - 12.4 так что проблем быть не должно. Архив собирается нормально, но в случае Xcode 15.3 при попытке залить архив в appstoreconnect получаю ошибку.

> Asset validation failed Invalid Bundle. The bundle <имя приложения>.app/Frameworks/MyTrackerSDK.framework does not support the minimum OS Version specified in the Info.plist.

Через Xcode 15.0 архив без проблем заливается в appstoreconnect.

**Comment by alysenko912** [comment]
Судя по данному [треду](https://developer.apple.com/forums/thread/749554), это косяк в Xcode 15.3. Воркэраунд на стороне MyTracker выглядел бы как установка MinimumOSVersion в Info.plist фреймворка в значение 100. Аналогично [поступили](https://github.com/firebase/firebase-ios-sdk/pull/12439/files#diff-f4eb4ff5ec89af999cbe8fa3ffe5647d7853ffbc9c1515b337ca043c684b6bb4R679) разработчики Firebase.

Но это странно, ведь с выходом нового Xcode, где данная проблема будет исправлена, нужен будет обратный фикс на стороне фреймворка (вместо 100 опять вернуть корректное значение).

Видимо, пока на Xcode 15.2 придется сидеть.

**Comment by aserrr** [comment]
> Судя по данному [треду](https://developer.apple.com/forums/thread/749554), это косяк в Xcode 15.3. Воркэраунд на стороне MyTracker выглядел бы как установка MinimumOSVersion в Info.plist фреймворка в значение 100. Аналогично [поступили](https://github.com/firebase/firebase-ios-sdk/pull/12439/files#diff-f4eb4ff5ec89af999cbe8fa3ffe5647d7853ffbc9c1515b337ca043c684b6bb4R679) разработчики Firebase.
> 
> Но это странно, ведь с выходом нового Xcode, где данная проблема будет исправлена, нужен будет обратный фикс на стороне фреймворка (вместо 100 опять вернуть корректное значение).
> 
> Видимо, пока на Xcode 15.2 придется сидеть.

Да, мы предварительно пришли к такому же выводу. Надёжнее и правильнее всего релизить из 15.2, так как в SDK проблем нет и не хотелось бы добавлять костыли. Но и такой вариант не исключаем пока. Гугл тоже 100.0 поставил.

**Comment by aserrr** [comment]
Проблема не воспроизводится в Xcode 15.4 beta
